department-of-veterans-affairs/abd-vro
# MCP-2127 Checkstyle fixes for app module

## What was the problem?
We had around 50 checkstyle warnings in the app module which needed to be fixed.

Associated tickets or Slack threads:

- [MCP-2127](https://amida.atlassian.net/browse/MCP-2127)

## How does this fix it?[^1]
Gets rid of checkstyle warnings in the build logs.

## How to test this PR
- Step 1 build project
- Step 2 verify that in the logs there are no checkstyle warnings for the app module


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
